docs/source/settings.rst

```diff
@@ -36,6 +36,19 @@ SPIDERMON_EXPRESSIONS_MONITOR_CLASS
 SPIDERMON_PERIODIC_MONITORS
 ---------------------------
 
+Default: ``{}``
+
+A dict containing the monitor suites that must be executed periodically as key and
+the time interval (in seconds) between the executions as value.
+
+For example, the following suite will be executed each 30 minutes:
```

Comment:
```suggestion
For example, the following suite will be executed every 30 minutes:
```

I feel the current wording is slightly wrong, but I’ve been unable to find a source supporting that claim, so feel free to leave it as is.